### ISSUE TITLE
cowsay

### ISSUE THREAD

**Issue by jeeftor** [tldr-pages/tldr#1265]
So the `cowsay` command allows you to use lots of "different" options for the command.  Is it appropriate to update the TLDR page to print out how you could get a list of the different `cowsay` options?  

I've used the following command: 

`ls /usr/share/cowsay -1 | rev | cut -c 5- | rev | xargs -I _ cowsay -f _ cowsay -f _`

to generate a list but didnt know if it fit into the overall mission of TLDR.  For example the mac `say` command has a TLDR option to print out the available voices.  Some commands, however, like this one dont have an internal feature for doing this but rather you have to look in a directory.

Thoughts?

```
 _______________
< cowsay -f tux >
 ---------------
   \
    \
        .--.
       |o_o |
       |:_/ |
      //   \ \
     (|     | )
    /'\_   _/`\
    \___)=(___/

 _________________
< cowsay -f udder >
 -----------------
  \
   \    (__)
        o o\
       ('') \---------
          \           \
           |          |\
           ||---(  )_|| *
           ||    UU  ||
           ==        ==
 _________________
< cowsay -f vader >
 -----------------
        \    ,-^-.
         \   !oYo!
          \ /./=\.\______
               ##        )\/\
                ||-----w||
                ||      ||

               Cowth Vader
 _______________________
< cowsay -f vader-koala >
 -----------------------
   \
    \        .
     .---.  //
    Y|o o|Y//
   /_(i=i)K/
   ~()~*~()~
    (_)-(_)

     Darth
     Vader
     koala
 _______________
< cowsay -f www >
 ---------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||--WWW |
                ||     ||
```

**Comment by waldyrious** [comment]
If there are 3 or 4 options, we can add them to the example description, or as multiple tokens separated by pipes. If there are many, then we could point out the usual location of the directory in the example description.

A similar case [happened with the assimp command](https://github.com/tldr-pages/tldr/pull/1028#issuecomment-257071872) and the discussion could be used as reference.

**Comment by waldyrious** [comment]
For future reference: this was closed in #1266. I have opened #1323 as I was unable to comment in that discussion.
